Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/OverlaySheetService.java

```diff
@@ -142,7 +148,74 @@ private void updateTagsView (SheetAdapter.SheetViewHolder holder, SheetModel mod
             String pickupChargesText = formattedPickupChargesText;
             String searchRequestId = model.getSearchRequestId();
             boolean showVariant =  !model.getRequestedVehicleVariant().equals(NO_VARIANT) && model.isDowngradeEnabled() && RideRequestUtils.handleVariant(holder, model, this);
-            if (model.getCustomerTip() > 0 || model.getDisabilityTag() || searchRequestId.equals(DUMMY_FROM_LOCATION) || model.isGotoTag() || showVariant || showSpecialLocationTag) {
+            if(model.getRideProductType().equals(RENTAL)){
+                holder.tagsBlock.setVisibility(View.VISIBLE);
+                holder.vcTierAndACView.setVisibility(View.GONE);
+                holder.reqButton.setTextColor(getColor(R.color.white));
+                holder.reqButton.setBackgroundTintList(ColorStateList.valueOf(getColor(R.color.turquoise)));
+                if (!variant.equals(NO_VARIANT) && key.equals("yatrisathiprovider")) {
```

Comment:
Why this variant change is required 
I Can't see it in design

Reply:
this is for yatrisathi specific to show ac and non ac